jupyter/d2l-java/chapter_convolutional-neural-networks/lenet.ipynb

```diff
@@ -180,49 +179,32 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 15,
+   "execution_count": 17,
    "metadata": {},
    "outputs": [],
    "source": [
     "NDManager manager = NDManager.newBaseManager();\n",
     "\n",
     "SequentialBlock block = new SequentialBlock();\n",
     "\n",
-    "block.add(\n",
-    "    Conv2D.builder()\n",
-    "            .setKernel(new Shape(5, 5))\n",
-    "            .optPad(new Shape(2, 2))\n",
-    "            .optBias(false)\n",
-    "            .setNumFilters(6).build())\n",
-    ".add(Activation::sigmoid)\n",
-    ".add(\n",
-    "    Pool\n",
-    "    .avgPool2DBlock(new Shape(5, 5), new Shape(2, 2), new Shape(2, 2)))\n",
-    ".add(\n",
-    "    Conv2D.builder()\n",
-    "            .setKernel(new Shape(5, 5))\n",
-    "            .setNumFilters(16).build())\n",
-    "    \n",
-    ".add(Activation::sigmoid)   \n",
-    ".add(\n",
-    "    Pool\n",
-    "    .avgPool2DBlock(new Shape(5, 5), new Shape(2, 2), new Shape(2, 2)))\n",
-    ".add(\n",
-    "    Linear.builder()\n",
-    "    .optFlatten(true)\n",
-    "    .setOutChannels(120)\n",
-    "    .build())\n",
-    ".add(Activation::sigmoid)\n",
-    ".add(\n",
-    "    Linear.builder()\n",
-    "    .setOutChannels(84)\n",
-    "    .build())\n",
-    ".add(Activation::sigmoid)\n",
-    ".add(\n",
-    "    Linear\n",
-    "    .builder()\n",
-    "    .setOutChannels(10)\n",
-    "    .build());\n",
+    "block.add(Conv2D.builder()\n",
```

Comment:
This still looks a bit off. The block level methods are at the same indentation level as the Conv2D level methods.